### PR TITLE
Add new lab meeting suggestions to agenda

### PR DESCRIPTION
**Pull request by KirstieJane**
Please review and let me know what you think!

### CODE REVIEW

**Comment by pherterich** [comment]
Works for me so far 👍

File: lab-meetings/README.md [hunk 1]
```diff
@@ -11,6 +11,49 @@ The lab also has a [google calendar](https://calendar.google.com/calendar/embed?
...
+| ---- | ----- | ----- | ------------ |
+| 5 December | No lab meeting | [Data Science for Mental Health Interest Group](https://turing-ds4mh.github.io/meetings.html) | |
+| 12 December | Ang | Data Visualisation | *Data Study Group so all remote meeting* (no rooms available) |
+| 19 December | Kirstie | Reflections on the year  | [2019-12-19-lab-meeting](https://whitakerlab.github.io/blog/2019-12-19-lab-meeting) |
+| 26 December | No lab meeting | Happy holidays 🎄☃️🎁 | *No blog* |
+| 2 January | No lab meeting | Happy new year 🎆🥂🎇 | *No blog* |
+| 9 January | Louise | New years resolutions | |
+| 16 January | Kirstie | Guest: František Váša | |
+| 23 January | Kirstie | Open agenda | |
+| 30 January | Ang & Yini | lunch: Chinese New Year :rat and Georgia speaking at [Data Science for Mental Health Interest Group](https://turing-ds4mh.github.io/meetings.html) | |
```
Comment (sgibson91) [inline lab-meetings/README.md]:
```suggestion
| 30 January | Ang & Yini | lunch: Chinese New Year :rat: and Georgia speaking at [Data Science for Mental Health Interest Group](https://turing-ds4mh.github.io/meetings.html) | |
```

Correcting the all important emoji!

**Comment by pherterich** [comment]
I think the session I'm supposed to lead on is now in the strike period - if we could switch that one with a different date that would be fab!

File: lab-meetings/README.md [hunk 2]
```diff
@@ -39,16 +39,16 @@ This table covers lab meetings from March 2020 to May 2020.
 | ---- | ----- | ----- | ------------ |
 | 5 March | TBD | Self care | |
```
Comment (malvikasharan) [inline lab-meetings/README.md]:
```suggestion
| 5 March | Malvika | Self care | [optional reading](https://greatist.com/happiness/ways-to-practice-self-care#1) |
```

**Comment by malvikasharan** [comment]
> I think the session I'm supposed to lead on is now in the strike period - if we could switch that one with a different date that would be fab!

@Patricia, the week after is no lab meeting, can you choose a slot that best suits you? I can chair the meeting next week as well on some other topic that I can ask the lab in this week's lab meeting.

**Comment by pherterich** [comment]
@malvikasharan 7 or 14 May maybe as we don't have leads allocated to these topics?

**Pull request merged**